### PR TITLE
Add command argument parser, add flags to pingsweep

### PR DESCRIPTION
**Pull request by KelsonBall**
This PR adds a `Commands` type that parses command line args into a dictionary, and then uses that parser to add optional functionality to the `pingsweep.py` script.

### CODE REVIEW

File: pingsweep.py [hunk 1]
```diff
@@ -1,5 +1,55 @@
...
+	description:
+		indicates start of sweep range
+	defaults to:
+		--from 1
+
+--to <number>
+	description:
+		indicates end of sweep range (non inclusive)
+	defaults to:
+		--to 255
```
Comment (Fallible) [inline pingsweep.py]: Looks like this needs to say 256.

File: pingsweep.py [hunk 2]
```diff
@@ -1,5 +1,55 @@
...
+	print(help_text)
+else:
+	# read from args or use default
+	ping_command = "ping -c 1"
+	if commands.has_flag("timeout"):
+		ping_command += " -W " + str(commands.int_of("timeout"))
+	ping_command += " " + (commands.string_of("address") or "10.11.1") + "."
+
+	start = commands.int_of("from") or 1
+	end = commands.int_of("to") or 255
```
Comment (Fallible) [inline pingsweep.py]: Increase to 256.

File: pingsweep.py (hunk 1 above)
Comment (KelsonBall) [reply pingsweep.py]: Instead of going to 256 I made it inclusive to match the expected behavior

**Review by Fallible — APPROVED** [review]
Approve the total rewrite of the initial script.

**Pull request merged**